runtime/j9vm/java11vmi.c

```diff
@@ -810,6 +812,16 @@ JVM_DefineModule(JNIEnv * env, jobject module, jboolean isOpen, jstring version,
 								clazz = vmFuncs->allClassesNextDo(&classWalkState);
 							}
 							vmFuncs->allClassesEndDo(&classWalkState);
+
+							Assert_SC_notNull(vm->anonClassLoader);
```

Comment:
I'd like to only do this second walk only if anon classes have been created.  We can check:
```
if (vm->anonClassCount > 0) {
   ......
}
```
and the locals related to this can be moved into this scope.

Reply:
Added the check.
@DanHeidinga please have another look.